app/src/main/java/com/distillery/interview/data/models/WeatherResponse.kt

```diff
@@ -0,0 +1,57 @@
+package com.distillery.interview.data.models
+
+import com.google.gson.annotations.SerializedName
+
+data class WeatherResponse(
+    @SerializedName("coord") val coord: Coord,
```

Comment:
`SerializedName` is only needed if you want to provide a different variable name than the name of the service, e.g.: @SerializedName("user_from_api") val user: User. So, in most of these cases `SerializedName` is not needed.

Reply:
Im agree with you! I only added it by coding convention! I always use @SerializedName in models due to obfuscation and to be free of GSON bugs in future.